### DISCUSSION TITLE
Allow object construction, maybe with `new`

### DISCUSSION THREAD

**Discussion by danfinlay** [endojs/Jessie#81] [Ideas]
Per this twitter thread:
https://twitter.com/marksammiller/status/1092866376006430720

I'm currently using `Reflect.construct(class, arguments)` as a workaround, but you mentioned this is also not valid Jessie, it merely happens to pass the Jessie linter.

**Comment by dckc** [comment]
To make this issue a bit more stand-alone without following the link: MarkM writes:
> the original rationale is that `new` is only about prototype inheritance, classes, or class-like patterns. All of these are only meaningful with inherited methods that mention `this`. Thus, all of these are absent from Jessie.
